core/testing/runner.odin

```diff
@@ -61,6 +63,18 @@ get_log_level :: #force_inline proc() -> runtime.Logger_Level {
 	}
 }
 
+JSON :: struct {
+	total:    int,
+	success:  int,
+	duration: time.Duration,
+	packages: map[string][dynamic]JSON_Test,
+}
+
+JSON_Test :: struct {
+	success: bool,
+	name:    string,
+}
+
```

Comment:
Because these two structs are never used outside of their particular `when` statement below, I would've also put them inside the `when` block itself. They're never used anywhere else in the program, and it's one less thing the compiler has to be concerned about. More importantly, it fences off all the possible places they could be used and makes understanding the scope a little easier.